darkmoon/api/v1/metadata/views.py

```diff
@@ -444,34 +447,16 @@ async def hash_comparison(
     try:
         inputFileType = str(fileInput.content_type)
         inputFileName = str(fileInput.filename)
-        data = fileInput.file.read()
-        h_md5 = hashlib.md5()  # noqa S324
-        h_sha1 = hashlib.sha1()  # noqa: S324
-        h_sha256 = hashlib.sha256()
-        h_sha512 = hashlib.sha512()
-        h_md5.update(data)
-        h_sha1.update(data)
-        h_sha256.update(data)
-        h_sha512.update(data)
-        md5Hash = h_md5.hexdigest()
-        sha1Hash = h_sha1.hexdigest()
-        sha256Hash = h_sha256.hexdigest()
-        sha512Hash = h_sha512.hexdigest()
-
-        obj = DocMetadataEntity(
-            _id=PydanticObjectId(),
-            name=[inputFileName],
-            file_type=[inputFileType],
-            operating_system=[""],
-            source_iso_name=[""],
-            file_extension=[""],
-            hashes=Hashes(
-                md5=md5Hash,
-                sha1=sha1Hash,
-                sha256=sha256Hash,
-                sha512=sha512Hash,
-            ),
-        )
+
+        tmp_path = Path("tmpfile")
```

Comment:
Please don't create your own implementation of the tempfile library. Please use the standard library version.